sktree/tests/test_honest_forest.py

```diff
@@ -263,6 +263,27 @@ def test_impute_posteriors(honest_prior, val):
         ), f"Failed with {honest_prior}, prior {clf.estimators_[0].empirical_prior_}"
 
 
+@pytest.mark.parametrize(
+    "honest_prior, val",
+    [
+        ("ignore", np.nan),
+    ],
+)
+def test_ignore_posteriors(honest_prior, val):
```

Comment:
```suggestion
def test_honestforest_predict_proba_with_honest_prior(honest_prior, val):
```

This test seems like it should conceptually test `predict_proba` for the HonestForestClassifier for different honest_priors?